draft-ietf-quic-transport.md

```diff
@@ -4571,10 +4565,14 @@ preferred_address (0x000d):
 +                                                               +
 |                                                               |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+| CID Length (8)|
```

Comment:
I think that this change is unnecessary, but you missed a varint.

Reply:
Connection ID lengths use a 1-byte encoding throughout the document (long headers, NEW_CONNECTION_ID). I'd rather keep this consistent.

Reply:
Ahh, right.  In consistency FTW.